jscomp/core/js_dump_property.ml

```diff
@@ -84,6 +84,7 @@ let property_access f s =
 let property_key (s : J.property_name) : string =
   match s with
   | Lit s ->
+      let s = Ext_ident.unwrap_uppercase_exotic s in
```

Comment:
Wondering whether `s` should have its own type other than string, so that one gets a reminder to do the unwrapping.
Here and in the other cases of ids.

Reply:
Good point, can be investigated separately.